### PR TITLE
Update JSON manifest

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
This is mainly useful for GNOME Builder.

### CODE REVIEW

File: al.getcyrst.jadegui.json [hunk 1]
```diff
@@ -1,38 +1,97 @@
...
+        "ls",
+        "pip3 install --prefix=/app ."
+      ]
+    },
+    {
+      "name": "jadegui",
+      "buildsystem": "meson",
+      "build-options": {
+        "build-args": [
+          "--share=network"
```
Comment (axtloss) [inline al.getcyrst.jadegui.json]: what is this needed for?

Comment (TheEvilSkeleton) [reply al.getcyrst.jadegui.json]: No idea. I got it from https://github.com/crystal-linux/jade-gui/blob/15ce03ee37a3ef6dd344553350d4aff08ad0a53d/al.getcryst.jadegui.yml#L80.

Comment (axtloss) [reply al.getcyrst.jadegui.json]: Oh weird, that was auto generated so i guess it has a reason to be there.

**Comment by axtloss** [comment]
I realized that the file name has a typo in it, it's `getcyrstal` instead of `getcrystal`, could you fix that?

**Review by axtloss — APPROVED** [review]

**Comment by axtloss** [comment]
Thank you for the contribution!

**Pull request merged**